src/i18n/en.json

```diff
@@ -293,6 +293,9 @@
         "data-portability": "Get data in a machine-readable format",
         "erase-all": "Erase all data",
         "erasure-data": "Data to erase",
+        "include-objection": "Additionally include an objection based on your particular situation",
+        "objection-reason": "Reason for your objection",
+        "objection-reason-explanation": "An objection has to be justified based on your particular situation. Explain why you specifically object to the company processing your data.",
```

Comment:
This definitely needs a more in depth explanation in a blog post. What are valid reasons for an objection? We should provide examples.

Reply:
And we should definitely tell users that it could just suffice to say that they just don't like their data being processed and that they don't need to invent a reason.

Reply:
I would love to do that but I don't think we can. The explanation is as in-depth as I can reliably provide. I haven't found any actual decisions on that topic and the legal commentary disagrees in important aspects.

> And we should definitely tell users that it could just suffice to say that they just don't like their data being processed and that they don't need to invent a reason.

That view is only supported by BeckOK, while others say that this doesn't suffice. *shrug*